src/arr.ts

```diff
@@ -211,53 +220,71 @@ function getRelevantArrInstances(mediaType: MediaType): string[] {
 			return sonarr ?? [];
 		case MediaType.MOVIE:
 			return radarr ?? [];
```

Comment:
can we make `sonarr` and `radarr` transform to an empty array at the zod layer so we don't have to do these null checks everywhere?

Reply:
yes, we can do that